examples/metrics/prometheus-install/strimzi-pod-monitor.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       strimzi.io/kind: cluster-operator
   namespaceSelector:
     matchNames:
-      - myproject
+      - my-project
```

Comment:
I'm not sure if this change was intended or not - but all the other example files use `myproject`. So I think we should try to keep it.

Reply:
gah this was my mistake. I'll correct it. and update the regex matching as requested below.